week-1/G-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `name`
-
-console.log(name);
+var name = "mariama";
+var namelength = name.length;
```

Comment:
can you use camel case naming convention to name namelength for example camelCase

Reply:
okay i will do that.